src/modules/trading-cockpit/components/FairValue/useFairValueStyles.tsx

```diff
@@ -0,0 +1,31 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+export const useFairValueStyles = makeStyles<Theme>((theme: Theme) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: '10px 16px',
+    background: `rgba(255,255,255,0.05)`,
+    borderRadius: '12px',
+  },
+
+  description: {
+    marginBottom: '6px',
+  },
+
+  descriptionTitle: {
+    fontSize: '14px',
+    lineHeight: '17px',
```

Comment:
please use relative value `1.2` or `1.215`
```suggestion
    lineHeight: 1.2,
```